EventFilter/Phase2TrackerRawToDigi/test/DTHDAQtoFEDRAWData_cfg.py

```diff
@@ -0,0 +1,54 @@
+# CMS job to run the DTHDAQToFEDRawDataConverter module
+# It converts .raw files containg orbits bitstream raw data from the TIF to FEDRawDataCollection
+import FWCore.ParameterSet.Config as cms
+
+process = cms.Process("FEDRAW")
+
+
+process.load("FWCore.MessageLogger.MessageLogger_cfi")
+
+# Set the logging output for debugging
+# process.MessageLogger.cerr.FwkReport.reportEvery = 1  # print every event
+# process.MessageLogger.cerr.threshold = 'INFO'
+
+process.MessageLogger = cms.Service(
+    "MessageLogger",
+    destinations=cms.untracked.vstring('logFile', 'cout'),
+    logFile=cms.untracked.PSet(
+        threshold=cms.untracked.string('INFO'),  # Change to 'DEBUG' if needed
+    ),
+    cout=cms.untracked.PSet(
+        threshold=cms.untracked.string('WARNING'),  # Only show warnings/errors on console
+    ),
+    categories=cms.untracked.vstring('DTHDAQToFEDRawDataConverter')
+)
+
+# Define an empty source because this is a producer that reads from a file
+process.source = cms.Source("EmptySource")
+
+# Limit the number of events processed (since we are reading from a file, 1 is fine)
```

Comment:
maxEvents determines the number of times that your produce() function will be called. If you only call it once, then you'll only write out a single event. We need to be able to process multiple events.